FWCore/Utilities/interface/CMSUnrollLoop.h

```diff
@@ -30,6 +30,12 @@
 #define CMS_UNROLL_LOOP_COUNT(N) _Pragma(STRINGIFY(clang loop unroll_count(N)))
 #define CMS_UNROLL_LOOP_DISABLE _Pragma(STRINGIFY(clang loop unroll(disable)))
 
+#elif defined(__INTEL_COMPILER)
+// Intel icc compiler
+#define CMS_UNROLL_LOOP _Pragma(STRINGIFY(unroll))
+#define CMS_UNROLL_LOOP_COUNT(N) _Pragma(STRINGIFY(unroll(N)))
+#define CMS_UNROLL_LOOP_DISABLE _Pragma(STRINGIFY(nounroll))
```

Comment:
This PR needs to be rebased (on top of CMSSW_12_4_X_2022-05-13-1100 or later) to account for changes made in #37876 (basically change `STRINGIFY` to `EDM_STRINGIZE`).

Reply:
Done. Rebased to head of CMSSW_12_4_X.